presto-verifier/src/main/java/com/facebook/presto/verifier/QueryResult.java

```diff
@@ -29,14 +29,16 @@
 
     private final State state;
     private final Exception exception;
-    private final Duration duration;
+    private final Duration latency;
```

Comment:
Lets name this `wallTime` instead of `latency`, since that's what we use more commonly in the rest of the code base